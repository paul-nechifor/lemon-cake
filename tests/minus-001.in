(- 1 1)
(- 100 10)
(- 5000 -6000)
(- -5000 6000)
(- -10000 -20020)
(- 123 99999)
