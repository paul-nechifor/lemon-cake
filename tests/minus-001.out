0
90
11000
-11000
10020
-99876
